static/js/ranking/ranking_page.tsx

```diff
@@ -371,8 +377,8 @@ class Page extends React.Component<RankingPagePropType, RankingPageStateType> {
           if (_.isEmpty(statData[place])) {
             continue;
           }
-          const placeStat = Object.values(statData[place])[0];
-          let value = _.isUndefined(placeStat.value) ? 0 : placeStat.value;
+          const placeStat = statData[place];
+          let value = placeStat.value || 0;
```

Comment:
I think we only set value to 0 if placeStat.value returns undefined because proto converts 0 to undefined. In the case of null, we probably don't want to set value to 0.

Reply:
I see. If it's null, then value will be null here, would that work? I changed back to the original behavior anyway.

Reply:
I just checked, yes, null will be handled. But also realized later code actually turns null into 0 anyways ..